### DISCUSSION TITLE
I want to translate Mobx homepage into korea language.

### DISCUSSION THREAD

**Discussion by kimyoungjae96** [mobxjs/mobx#3020] [General] [answered]
Hello, I am an employee living in Korea who works in a mobility service. I contact you because I had so many good experiences using Mobx and wanted to explain it to people in our country in a more friendly way. In other words, I want to translate the contents of the Mobx homepage into our language and give valuable work to many people, can you tell me how to contribute?

**Comment by iChenLei** [comment, marked as answer]
Same with https://github.com/mobxjs/mobx/discussions/2781, Anyway thanks for your enthusiasm. 

Append: https://github.com/mobxjs/zh.mobx.js.org is a good example for you, You are korean so you need a domain named `https://kr.mobx.js.org`, register the subdomain from github.com/js-org/js.org (e.g. https://github.com/js-org/js.org/pull/5604). And you can host it on netlify / vercel or other free host services. (zh.mobx.js.org hosted on vercel.com
> If you need more guide, please @ me and write clear question

**Reply by kimyoungjae96** [reply]
Thank you very much for your help. I hope you have a great day. :)

**Reply by kimyoungjae96** [reply]
@iChenLei 
Can we add mobxjs/ko.mobx.js.org? 
If it's hard, is there any other way to recommend it?

**Reply by iChenLei** [reply]
Ok, If you really want to do the boring and hard translation work, I will help you to build website CI/CD and translation workflow and more work (exclude korean translation). **Documentation translation is not a easy work!!**
> I think `kr.mobx.js.org` is more  meaningful ? Same with vuejs korean documentation https://kr.vuejs.org/index.html

**Reply by kimyoungjae96** [reply]
I think `kr.mobx.js.org` is more  meaningful ? Same with vuejs korean documentation https://kr.vuejs.org/index.html
> I agree.

I think I'll learn many things from translation even though it's hard and boring. so I want to take responsibility until the end. also, I'm actively looking for team members to work with.

In other words, I really want to translate. so, I would appreciate it if you could tell me how to proceed from now on.

**Reply by iChenLei** [reply]
@kimyoungjae96 👍   for your enthusiasm. I will do prepare work(website auto deploy, init repository and more) for `kr.mobx.js.org` this weekend. Before all work ready you can keep an eye on `zh.mobx.js.org`'s PR Label, CI/CD flow and other useful information, it will help you become a better korean translation organizer.
> Useful link
[discussions/2781](https://github.com/mobxjs/mobx/discussions/2781)
> https://github.com/mobxjs/zh.mobx.js.org

**Reply by kimyoungjae96** [reply]
okay. thank you 👍

**Reply by kimyoungjae96** [reply]
I did a some research on the domain and I think ko.mobx.js.org is better than kr.mobx.js.org
ex) https://ko.reactjs.org/, https://v3.ko.vuejs.org/, https://ko.javascript.info/

**Reply by iChenLei** [reply]
OK, `ko.mobx.js.org` is better.

**Reply by iChenLei** [reply]
@kimyoungjae96 Hi, [ko.mobx.js.org](https://github.com/mobxjs/ko.mobx.js.org) project is available now, you can create PR to design Korean translation workflow (and then @ me to merge) ! It auto deploy to vercel platform (preview link is https://ko-mobx-js-org.vercel.app). `ko.mobx.js.org` domain registe is my work and you don't need care it. Thank u very much, handsome korea boy.

**Reply by kimyoungjae96** [reply]
wow, awesome!! Thank you so much for your help!

**Reply by kimyoungjae96** [reply]
Could you please add wiki? I just want to add best translation practices to wiki.

**Reply by iChenLei** [reply]
@kimyoungjae96 Done, but i think you can add korean translation workflow guide to README.md via `PR` directly.

**Reply by kimyoungjae96** [reply]
okay. don't worry. I will explain the korean translation workflow using readme.md and wiki together.
